arrows/pytorch/grid.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import torch
+
+
+class grid(object):
+    def __init__(self, grid_row=15, grid_cols=15, target_neighborhood_w=7):
+        self._grid_rows = grid_row
+        self._grid_cols = grid_cols
+        self._target_neighborhood_w = target_neighborhood_w
+        self._half_cell_w = int(self._target_neighborhood_w // 2)
+
+    def __call__(self, im_size ,bbox_list, MOT_flag=False):
+        return self.obtainGridFeatureList(im_size, bbox_list, MOT_flag)
+
+    @property
+    def img_h(self):
+        return self._img_h
+
+    @img_h.setter
+    def img_h(self, val):
+        self._img_h = val
+
+    @property
+    def img_w(self):
+        return self._img_w
+
+    @img_w.setter
+    def img_w(self, val):
+        self._img_w = val
+    
+    def obtainGridFeatureList(self, im_size, bbox_list, MOT_flag):
+        r"""
+            The output of the function is a grid feature list for each corresponding bbox of current frame/image
+        """
+        self._img_w, self._img_h = im_size
+
+        # calculate grid cell height and width
+        cell_h = self._img_h / self._grid_rows
+        cell_w = self._img_w / self._grid_cols
+
+        # initial all gridcell to 0
+        grid = torch.FloatTensor(self._grid_rows, self._grid_cols).zero_()
+
+        bbox_id_centerIDX = {}
+        # build the grid for current image
+        for idx, item in enumerate(bbox_list):
+            if MOT_flag is True:
+                bb = item
+            else:
+                bb = item.bounding_box()
```

Comment:
Could the test just be `if MOT_flag:`? This would also still be readable as one line:
```python
bb = item if MOT_flag else item.bounding_box()
```